extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/services.js

```diff
@@ -137,6 +137,9 @@ angular.module('PaperUI.services', [ 'PaperUI.constants' ]).config(function($htt
                     } else if (parameter.context.toUpperCase() === 'DAYOFWEEK') {
                         parameter.element = 'dayofweek';
                         parameter.inputType = 'text';
+                    } else if (parameter.context.toUpperCase() === 'PASSWORD') {
+                        parameter.element = 'input';
+                        parameter.inputType = parameter.context === 'password' ? 'password' : 'text';
```

Comment:
If the context in upper case is equal to PASSWORD, why not set the inputType set to 'password' every time?
Why to we need to differ between
* context = "Password" => inputType = text
* context = "password" => inputType = password